lib/libutee/tee_api_property.c

```diff
@@ -131,6 +118,56 @@ static bool is_propset_pseudo_handle(TEE_PropSetHandle h)
 	       h == TEE_PROPSET_TEE_IMPLEMENTATION;
 }
 
+static TEE_Result get_core_property(unsigned long prop_set, unsigned long idx,
+				    void *name, uint32_t *name_len, void *buf,
+				    uint32_t *blen, uint32_t *prop_type)
+{
+	TEE_Result res = TEE_SUCCESS;
+	uint32_t bl = 0;
+
+	if (!blen)
+		return _utee_get_property(prop_set, idx, name, name_len, buf,
+					  NULL, prop_type);
+
+	bl = *blen;
+	res = _utee_get_property(prop_set, idx, name, name_len, buf, &bl,
+				 prop_type);
+	/*
+	 * Binary blocks are returned as raw binary data, while the TA
+	 * expects it as base64 coded data. So we need to translate it.
+	 */
+
```

Comment:
minor: It seems to me the empty line would better fit above the comment rather than below.